From: Thomas Renninger <trenn@suse.de>
Subject: ACPI: Overriding ACPI tables via initrd only works with an initrd
References: none
Patch-Mainline: Submitted

Reflect this dependency in Kconfig.
Shorten the config description as suggested by Borislav Petkov

Signed-off-by: Thomas Renninger <trenn@suse.de>
---
 drivers/acpi/Kconfig |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/drivers/acpi/Kconfig b/drivers/acpi/Kconfig
index 44cea5d..ca939b4 100644
--- a/drivers/acpi/Kconfig
+++ b/drivers/acpi/Kconfig
@@ -262,7 +262,8 @@ config ACPI_CUSTOM_DSDT
 	default ACPI_CUSTOM_DSDT_FILE != ""
 
 config ACPI_INITRD_TABLE_OVERRIDE
-	bool "ACPI tables can be passed via uncompressed cpio in initrd"
+	bool "ACPI tables override via initrd"
+	depends on BLK_DEV_INITRD
 	default n
 	help
 	  This option provides functionality to override arbitrary ACPI tables
